core/point/point.go

```diff
@@ -21,3 +25,34 @@ func (p *Point) X() int64 { return p.x }
 
 // Y returns the y-value.
 func (p *Point) Y() int64 { return p.y }
+
+// Kilo Foxtrot addition below
+// Three point translation methods, one is an alternate
+
+// 1. '*Point' immutable int point, to SGF(String of two letters) Point
+func translateToSGFPt01(pt *Point) string {
```

Comment:
How about we make this a method on the Point object

func (pt *point) ToSGF() string {

}